examples/persons/src/Client.re

```diff
@@ -6,5 +6,31 @@ let httpLink =
     (),
   );
 
+/* WebSocket client */
+let webSocketLink =
+  ApolloLinks.webSocketLink(
+    ~uri="wss://subscriptions.graph.cool/v1/cjdgba1jw4ggk0185ig4bhpsn",
+    ~reconnect=true,
+    (),
+  );
+
+/* based on test, execute left or right */
```

Comment:
left or right you mean the ternary logic?

Reply:
My guess it was copypasted from [reason-apollo examples](https://github.com/apollographql/reason-apollo/blob/master/examples/swapi/src/Client.re#L45), not sure what it means :smile:
I've replaced it with the comment about link splitting from the official [apollo docs](https://www.apollographql.com/docs/react/data/subscriptions/#client-setup).